controllers/topics.controller.js

```diff
@@ -0,0 +1,14 @@
+
+const {fetchTopics} = require('../models/topics.models')
+
+
+
+exports.getTopics =  (req, res) =>{
+    
+    fetchTopics()
+    .then((Data)=>{
+        res.status(200).send(Data);
+       
+    })
```

Comment:
You will need to change something here to ensure you are sending an object with a key of topics. `Data` isn't a great naming convention here. `topics` is a lot better 😄